stacks/product-jenkins/versions.tf

```diff
@@ -1,4 +1,10 @@
 
 terraform {
   required_version = ">= 0.12"
+  required_providers {
```

Comment:
I am not sure if this will work.  this is a terraform 0.13 feature, and the product operator that consumes this terraform still uses 0.12

Reply:
Changed it to >= 0.13

Reply:
that won't actually solve the problem, it'll actually make it worse - the container the product operator uses doesn't have that version of tf installed

Reply:
https://www.terraform.io/docs/language/providers/requirements.html - `required_providers` was added in terraform 0.13.  so if we're still using 0.12, I would expect this to error out immediately.

Reply:
let's just remove `required_providers` entirely since it is dead code anyways.  we can add it when we update to 0.13